models/customer/src/types.ts

```diff
@@ -1,7 +1,13 @@
 import type { Customer, CustomerDraft } from '@commercetools/platform-sdk';
+import { TAddress } from '@commercetools-test-data/commons';
 import type { TBuilder } from '@commercetools-test-data/core';
 
-export type TCustomer = Customer;
+export type TCustomer = Customer & {
+  defaultShippingAddress: TAddress | null;
+  shippingAddresses: TAddress[] | null;
+  defaultBillingAddress: TAddress | null;
+  billingAddresses: TAddress[] | null;
+};
 export type TCustomerDraft = CustomerDraft;
```

Comment:
These new properties that we're adding in the `TCustomer` type will only exist in the GraphQL representation of this model but we need to add them here so consumers can mock them. Is this statement right?